common/namespace/cache.go

```diff
@@ -631,34 +571,6 @@ func (entry *CacheEntry) Clone(ms ...EntryMutation) *CacheEntry {
 	return FromPersistentState(entry.thisCluster, &r)
 }
 
-func (entry *CacheEntry) duplicate() *CacheEntry {
```

Comment:
I only know this because I'm debugging the 1.11.4 -> 1.12.0 GC behavior but FYI this function alone represents about 5% of all allocations in the matching service.